Atomic/syscontainers.py

```diff
@@ -184,7 +184,10 @@ def install(self, image, name):
 
     def _check_oci_configuration_file(self, conf_path, remote=None):
         with open(conf_path, 'r') as conf:
-            configuration = json.loads(conf.read())
+            try:
+                configuration = json.loads(conf.read())
+            except ValueError:
+                raise ValueError("Invalid json in configuration file.")
```

Comment:
I understand your point, but If I was using the atomic tool, and got this error message, I would have no idea what it means.  Which configuration file would be my question, and I could potentially waste time searching and looking at any configuration file related to atomic command.  /etc/atomic.conf, /etc/atomic.d/* /etc/containers ...